components/classes/b2b/account-team.schema.json

```diff
@@ -0,0 +1,39 @@
+{
+  "meta:license": [
+    "Copyright 2025 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/account-team",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "B2B Account Team",
+  "type": "object",
+  "description": "A mapping between the B2B Account owning team key and one or more user keys for users in B2B Sales User.",
+  "meta:extends": ["https://ns.adobe.com/xdm/data/record"],
```

Comment:
Also please add the following line after `meta:extends`
`"meta:abstract": true,`